Library/Homebrew/keg_relocate.rb

```diff
@@ -87,16 +87,18 @@ def change_rpath(file, new_prefix)
     end
     return unless patchelf.installed?
     cmd = "#{patchelf.bin}/patchelf --set-rpath #{new_prefix}/lib"
-    (old_i, old_p) = `#{patchelf.bin}/patchelf --print-rpath --print-interpreter #{file}`.split("\n")
+    old_p = `#{patchelf.bin}/patchelf --print-rpath #{file}`.split("\n")
+    old_i = ""
     raise ErrorDuringExecution.new(cmd) unless $?.success?
     if file.mach_o_executable?
+      old_i = `#{patchelf.bin}/patchelf --print-interpreter #{file}`.split("\n")
       interpreter = new_prefix == PREFIX_PLACEHOLDER ?
         "/lib64/ld-linux-x86-64.so.2" :
         HOMEBREW_PREFIX/"lib/ld.so"
```

Comment:
Change this to `"#{HOMEBREW_PREFIX}/lib/ld.so"
so that it's a string both ways, rather than a `String` and a `Pathname`.